src/runtime/runtime_unix.go

```diff
@@ -83,3 +78,10 @@ func ticks() timeUnit {
 func syscall_Exit(code int) {
 	exit(code)
 }
+
+func extalloc(size uintptr) unsafe.Pointer {
+	return malloc(size)
```

Comment:
To consider: you could use `calloc` instead and avoid zeroing the memory. In some cases this might be faster or use less memory. For example, for large allocations on Linux the libc will likely return a new `mmap` area that has been zeroed by the OS already (and might not even have RAM to back it).

Reply:
I am going to leave this, because I do not want to impose any constraints on any future external allocators.